src/blocks/icons/IconWrapper.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, ReactNode } from 'react';
+import styled, { FlattenSimpleInterpolation } from 'styled-components';
+import { getBlocksColor } from '../Blocks.utils';
+import { IconProps } from './Icons.types';
+
+type IconWrapperProps = IconProps & {
```

Comment:
Can omit `ref` and `as` props to get rid of the type error
`type IconWrapperProps = Omit<IconProps, 'as' | 'ref'> & {
`